packages/ui/src/number-field/NumberField.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import { UseState } from '@superdispatch/ui-docs';
+import { NumberField } from '..';
+
+export default { title: 'Inputs/NumberField', component: NumberField };
+
+export const basic = () => (
+  <UseState initialState={false}>
+    {(value, setValue) => (
+      <NumberField
+        value={value}
```

Comment:
@tokarevfedor1991 Is it ok to use false as a default value for NumberField?

Reply:
we can set false, but there will be nothing, not even 0

Reply:
I'm thinking about situations when we will refer to these stories to copy/paste the code. And to make it clearer for future users, we can set the default value that we are using for this field in Carrier TMS.

Reply:
I will set '0' as default

Reply:
Can you check what is used in Carrier TMS? But I think it's not critical=)